docs/src/tutorial.md

```diff
@@ -159,8 +159,8 @@ Ah, a scientific discovery: Setosa has short but wide sepals!
 
 Color scales in Gadfly by default are produced from perceptually uniform
 colorspaces (LUV/LCHuv or LAB/LCHab), though it supports RGB, HSV, HLS, XYZ, and
-converts arbitrarily between these. Of course, CSS/X11 named colors work too:
-"old lace", anyone?
+converts arbitrarily between these. Color values can also be specified by most names common to CSS or X11, e.g. `"oldlace"` or `"aliceblue"`. The [full list of valid color names](http://juliagraphics.github.io/Colors.jl/v0.9/namedcolors/) is defined in the [Colors.jl library](http://juliagraphics.github.io/Colors.jl/v0.9/).
```

Comment:
Changing the links to e.g. http://juliagraphics.github.io/Colors.jl/stable/namedcolors/ would keep the links up to date with the latest tagged version of Colors (currently v0.11).

Reply:
agree; i'll edit and resubmit

Reply:
No rush, it's good to wait for other comments, and generally Gadfly PRs are left open for several days.